x-pack/plugins/fleet/public/applications/fleet/sections/agents/components/agent_upgrade_modal/constants.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// Available versions for the upgrade of the Elastic Agent
+// These versions are only intended to be used as a fallback
+// in the event that the updated versions cannot be retrieved from the endpoint
+
+export const FALLBACK_VERSIONS = [
```

Comment:
Should we add 8.3.0 too here?  or can we dynamically add the current kibana version so it's not breaking when we create a new kibana version?

Reply:
Added the version dynamically in the component.